recipes/molearn/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "molearn" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Degiacomi-Lab/{{ name }}/archive/v{{ version }}.tar.gz
```

```suggestion
  url: https://github.com/Degiacomi-Lab/molearn/archive/v{{ version }}.tar.gz
```
This can confuse our bots.

---

This is exactly how I have done previous recipes, and how I was recommended to do recipes. It's never confused the bots before.

---

I've also put this question to others and the only response was simple templating like this is fine.

---

Yes for the PyPI URL that is correct but I have seen these issues for GitHub.
Additionally, anything below (import, doc URL) should definitely be changed. As you can see in the gitter convo, it doesn't add anything but just increases compute time and makes recipes non-standard.